src/pages/waiting-room-to-car/cat-c/components/vehicle-checks-modal/vehicle-checks-modal.cat-c.page.ts

```diff
@@ -185,10 +185,23 @@ export class VehicleChecksCatCModal {
     this.store$.dispatch(new TellMeQuestionOutcomeChanged(result, index));
   }
 
-  shouldDisplayBanner = (): boolean => {
+  nonTrailerBanner() {
     return (
       this.vehicleChecksScore.drivingFaults === 4 &&
-      this.vehicleChecksScore.seriousFaults === 1
+       this.vehicleChecksScore.seriousFaults === 1 &&
```

Comment:
indentation might cause the build to fail

Reply:
also the name of the function should be more specific about the action that it does. And please add a return type to it